### PR TITLE
ScreenLayouts changed, Google Calendar Integration Updadted with Goog…

### PR DESCRIPTION
**Pull request by Huzii-droid19**
New Task Screen Layout Changed, with the change of NewTask Screen, Some of its components props are also changed
Google Meet Integration with Google Calendar when creating a new task is done.

### CODE REVIEW

File: src/components/MeetupButton/index.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  const [meetUpText, setMeetUpText] = React.useState<string>(
+    'Add video conference',
+  );
+  const handleAddMeetUp = () => {
+    setMeet(true);
+    setMeetIconName('video-outline');
+    setMeetUpText('Video conference added');
+  };
+  const handleRemoveMeetUp = () => {
+    setMeet(false);
```
Comment (shoaib2529) [inline src/components/MeetupButton/index.tsx:23]: Don't use 3 states for it as all of them are changing at the same time. So use single state with object may be.

**Review by shoaib2529 — CHANGES_REQUESTED** [review]
Also I can't see where we are adding meet link in todos, is it pending?

File: src/components/MeetupButton/index.tsx (hunk 1 above)
Comment (Huzii-droid19) [reply src/components/MeetupButton/index.tsx:23]: ok

Comment (Huzii-droid19) [reply src/components/MeetupButton/index.tsx]: done

**Pull request merged**